devops/openshift/db-migrations-job.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: template.openshift.io/v1
+kind: Template
```

Comment:
Correct me if I am wrong, why is the kind given here as template and not job?

The reference from user story shows a sample job specification like this?

![image](https://github.com/bcgov/SIMS/assets/54600590/01848adb-5e39-490b-b0d9-9913f2b9e7d5)

Reply:
The template contains the job, a template commonly used to encapsulates the necessary configuration parameters and objects, making it easier to be used in the job or the deployment configs consistently.
![image](https://github.com/bcgov/SIMS/assets/62901416/27984d07-c69a-4fb8-8983-f1b3d3cd9b8d)